app/screens/price/price-history-screen.tsx

```diff
@@ -2,16 +2,60 @@ import { makeStyles } from "@rneui/themed"
 import * as React from "react"
 import { PriceHistory } from "../../components/price-history"
 import { Screen } from "../../components/screen"
+import { GaloyPrimaryButton } from "@app/components/atomic/galoy-primary-button"
+import { useNavigation } from "@react-navigation/native"
+import { StackNavigationProp } from "@react-navigation/stack"
+import { RootStackParamList } from "@app/navigation/stack-param-lists"
+import { usePriceHistoryScreenQuery } from "@app/graphql/generated"
+import { gql } from "@apollo/client"
+import { useLevel } from "@app/graphql/level-context"
+import { useI18nContext } from "@app/i18n/i18n-react"
 
 const useStyles = makeStyles((_theme) => ({
   screen: { flex: 1 },
+  button: { padding: 20 },
 }))
 
+gql`
+  query priceHistoryScreen {
+    me {
+      id
+      defaultAccount {
+        id
+      }
+    }
+  }
+`
+
 export const PriceHistoryScreen: React.FC = () => {
+  const { navigate } = useNavigation<StackNavigationProp<RootStackParamList, "Primary">>()
+  const { LL } = useI18nContext()
+
+  const { data } = usePriceHistoryScreenQuery()
+  const accountId = data?.me?.defaultAccount?.id
+
+  const { isAtLeastLevelTwo } = useLevel()
+
+  const [urlWebView, setUrlWebView] = React.useState("https://fiat.blink.sv")
+  React.useEffect(() => {
+    setUrlWebView(`https://fiat.blink.sv?accountId=${accountId}`)
+  }, [accountId])
+
   const styles = useStyles()
   return (
     <Screen preset="scroll" style={styles.screen}>
       <PriceHistory />
+      {isAtLeastLevelTwo && (
+        <GaloyPrimaryButton
+          title={LL.PriceHistoryScreen.buyAndSell()}
+          onPress={() =>
+            navigate("webViewDebug", {
+              url: urlWebView,
+            })
+          }
+          style={styles.button}
```

Comment:
If I remember correctly during the demo the button was flashing black when pressed and i think it is because this line should be `containerStyle={styles.button}`.

Reply:
yup thanks